src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Alerts can be created by users with admin permission or higher from **Project Settings > Alerts > New Alert**.
+
+There are two types of alerts: Issue Alerts and Metric Alerts. 
+
+## Metric Alerts
+
+Metric alerts allow you to filter and set thresholds on all errors across a project. They can be used for high-level monitoring of patterns, or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors, or volume of database errors.
+
+### Alert Builder
+
+[ builder image ]
+
+A metric alert can have at most two triggers. The first one is a (required) critical trigger, and the second one is an (optional) warning trigger. Triggers are independent with one constraint: the warning trigger must be reached before the critical one.
+
+Triggers are evaluated roughly every minute from the highest severity to lowest. If there's a match, an alert is created with that status. Or, if an alert exists, its status is updated. A metric alert can be resolved manually, or automatically by setting resolution thresholds on it.
+
+When an alert is created or changes status, the actions associated with the trigger are executed. The available actions are:
+
+- Send an email (to a member or team). If sent to a member, the member's personal project alert opt-out settings are overridden.
+- Send a Slack notification
+
+### Alert Stream
+
+Alerts can be accessed via the **Alerts** sidebar item.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Untitled.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Untitled.png)
+
+### Alert Details
+
+This page shows you details about **one instance** of an alert. For example, from the time it started to the time it was resolved (or the current time if it's active).
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/03_Alert_Details.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/03_Alert_Details.png)
+
+The "subscribe" button subscribes you to workflow notifications for the alert, which include status changes and comments.
+
+## Issue Alerts
+
+Issue alerts fire whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users.
+
+### **Conditions**
+
+Conditions are evaluated for an issue **each time the issue receives a new event**.
+
+**Each condition is evaluated independently of other conditions**. For example, the following alert will never fire:
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_21.52.38.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_21.52.38.png)
+
+This is because an event can't cause both of the following conditions to be satisfied simultaneously: 
```

Comment:
This should be followed by a noun or it's an unclear reference. Perhaps, "In the example, the alert will not fire because an event cannot satisfy both of these conditions simultaneously:"